pkg/release/release.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package release
 
-var Base = "4.11.5"
+var Base = "4.11.0-0.nightly-2022-09-30-015126"
```

Comment:
What is the output of `oc get configmap/release-release-images-latest -o yaml`? Should we get only images accepted in https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/?

Reply:
`4.11.0-0.nightly-2022-09-30-015126` is accepted, just in a [nightly stream](https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/#4.11.0-0.nightly). I believe we wanted to rebase regularly on nightlies, not on Z streams. This is configured [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml#L104).

Reply:
Final rebase to a Z-stream will be done manually?

Reply:
We'll start with nightly rebase, but then we'll have to also implement Z-stream after the initial release.